src/core/echarts.ts

```diff
@@ -418,6 +422,8 @@ class ECharts extends Eventful<ECEventDefinition> {
 
             defaultRenderer = root.__ECHARTS__DEFAULT__RENDERER__ || defaultRenderer;
 
+            defaultCoarsePointer = root.__ECHARTS__DEFAULT__COARSE_POINTER || defaultCoarsePointer;
```

Comment:
If `root.__ECHARTS__DEFAULT__COARSE_POINTER` is `false`, the default coarse pointer mode still fallbacks to `auto`.